### PR TITLE
add stdsim and stdstats

### PR DESCRIPTION
**Pull request by japhir**
This project shares all the more general functions as an R package, as well as the application of this code for this particular paper in a separate repository. The code itself is general enough that it may be useful to others, and hopefully the repository is set up as a good example.

### CODE REVIEW

**Comment by japhir** [comment]
and it is shameless self-promotion ;-)

File: README.md [hunk 1]
```diff
@@ -43,6 +43,7 @@ A curated list of awesome research code, software, manuals, and more on Git, dev
 - [PuReGoMe](https://github.com/puregome/notebooks) - Notebooks of the PuReGoMe Project of the Netherlands eScience Center and Utrecht University.
 - [Saliency-Tubes-Visual-Explanations-for-Spatio-Temporal-Convolutions](https://github.com/alexandrosstergiou/Saliency-Tubes-Visual-Explanations-for-Spatio-Temporal-Convolutions) - Implementation of Saliency Tubes for 3D Convolutions in Pytoch and Keras to localise the focus spatio-temporal regions of 3D CNNs.
 - [SoftPool](https://github.com/alexandrosstergiou/SoftPool) - Code for approximated exponential maximum pooling.
+- [stdsim](https://github.com/japhir/stdsim/) - R package for Simulating Clumped Isotope standard statistics, associated [stdstats](https://github.com/japhir/stdstats) code that accompanies the paper:Kocken, I. J., Müller, I. A., & Ziegler, M. (2019). Optimizing the use of carbonate standards to minimize uncertainties in clumped isotope data. Geochemistry, Geophysics, Geosystems, 20, 5565– 5577. https://doi.org/10.1029/2019GC008545
```
Comment (kequach) [inline README.md]:
```suggestion
[stdstats](https://github.com/japhir/stdstats) - Simulation and plotting code for "Optimizing the use of carbonate standards to minimize uncertainties in clumped isotope data"
```
We would suggest to add stdstats instead after a license has been added. We opened an issue in the corresponding repository.
Optionally, also look at a possible Zenodo publication https://docs.github.com/en/repositories/archiving-a-github-repository/referencing-and-citing-content

Thank you for the contribution!

**Review by kequach — CHANGES_REQUESTED** [review]

**Comment by japhir** [comment]
I've added the license and accepted the changes :)

File: README.md [hunk 2]
```diff
@@ -43,6 +43,7 @@ A curated list of awesome research code, software, manuals, and more on Git, dev
 - [PuReGoMe](https://github.com/puregome/notebooks) - Notebooks of the PuReGoMe Project of the Netherlands eScience Center and Utrecht University.
 - [Saliency-Tubes-Visual-Explanations-for-Spatio-Temporal-Convolutions](https://github.com/alexandrosstergiou/Saliency-Tubes-Visual-Explanations-for-Spatio-Temporal-Convolutions) - Implementation of Saliency Tubes for 3D Convolutions in Pytoch and Keras to localise the focus spatio-temporal regions of 3D CNNs.
 - [SoftPool](https://github.com/alexandrosstergiou/SoftPool) - Code for approximated exponential maximum pooling.
+[stdstats](https://github.com/japhir/stdstats) - Simulation and plotting code for "Optimizing the use of carbonate standards to minimize uncertainties in clumped isotope data"
```
Comment (kequach) [inline README.md]:
```suggestion
- [stdstats](https://github.com/japhir/stdstats) - Simulation and plotting code for "Optimizing the use of carbonate standards to minimize uncertainties in clumped isotope data"
```
My bad, I missed the list symbol. After this change it's good to merge!

**Review by kequach — CHANGES_REQUESTED** [review]

**Review by kequach — APPROVED** [review]

**Pull request merged**